pandas/tseries/period.py

```diff
@@ -271,9 +271,10 @@ def _from_arraylike(cls, data, freq, tz):
     @classmethod
     def _simple_new(cls, values, name=None, freq=None, **kwargs):
         if not getattr(values, 'dtype', None):
-            values = np.array(values, copy=False)
-        if is_object_dtype(values):
-            return PeriodIndex(values, name=name, freq=freq, **kwargs)
+            try:  # assume values / ints
+                values = np.array(values, copy=False, dtype='int64')
+            except TypeError:  # objects / Periods
```

Comment:
I just wondering whether the following behavior (on current master) are the same. Also is no consistencies between list and ``ndarray``.

```
pd.PeriodIndex._simple_new([pd.Period('2011-01', freq='M')])
# PeriodIndex(['2011-01'], dtype='int64', freq='M')

pd.PeriodIndex._simple_new(np.array([pd.Period('2011-01', freq='M')]))
# PeriodIndex(['2011-01'], dtype='int64', freq='M')
```

```
pd.PeriodIndex._simple_new([1.1], freq='M')
# ValueError: Ordinal must be an integer

pd.PeriodIndex._simple_new(np.array([1.1]), freq='M')
# ValueError: Ordinal must be an integer
```